FreeRTOS/FreeRTOS-Kernel
# Update xSemaphoreGetStaticBuffer prototype in comment

Description
-----------
* Update xSemaphoreGetStaticBuffer prototype in comment

Test Steps
-----------
No code change in this PR.

Checklist:
----------

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
